model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java

```diff
@@ -1554,6 +1595,18 @@ public void setDockerAuthenticationFlow(AuthenticationFlowModel flow) {
         realm.setDockerAuthenticationFlow(flow.getId());
     }
 
+    @Override
+    public AuthenticationFlowModel getCIBAFlow() {
+        String flowId = getAttribute(RealmAttributes.CIBA_AUTHENTICATION_FLOW);
+        if (flowId == null) return null;
+        return getAuthenticationFlowById(flowId);
+    }
+
+    @Override
+    public void setCIBAFlow(AuthenticationFlowModel flow) {
```

Comment:
As mentioned before. We can move the `cibaFlow` to `CIBAPolicy`. No need for this method.

Reply:
@pedroigor Thank you for the clarification. I will put `cibaFlow` onto `CIBAPolicy`.